testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import print_function
 import torch
-from torch.autograd import Variable
+from torch.autograd import Variable, Function
```

Comment:
Why import `Function` is needed?

Reply:
For this class:
https://github.com/opencv/opencv_extra/blob/052200ca49780a922cac144d69852021897a47d2/testdata/dnn/onnx/generate_onnx_models.py#L609-L616